Traceback (most recent call last):
  File "main.py", line 203, in <module>
    lr=lr_pre, patch_size=patch_size, pretrained='models/pretrained_G_step225000')
  File "/raven/u/lumah/code/E6040-super-resolution-project/training_pre.py", line 46, in training_pre
    model.load_state_dict(torch.load(pretrained))
  File "/mpcdf/soft/SLE_15/packages/skylake/pytorch/gpu-cuda-11.2/anaconda_3_2020.02-2020.02/1.8.1/lib/python3.7/site-packages/torch/serialization.py", line 579, in load
    with _open_file_like(f, 'rb') as opened_file:
  File "/mpcdf/soft/SLE_15/packages/skylake/pytorch/gpu-cuda-11.2/anaconda_3_2020.02-2020.02/1.8.1/lib/python3.7/site-packages/torch/serialization.py", line 230, in _open_file_like
    return _open_file(name_or_buffer, mode)
  File "/mpcdf/soft/SLE_15/packages/skylake/pytorch/gpu-cuda-11.2/anaconda_3_2020.02-2020.02/1.8.1/lib/python3.7/site-packages/torch/serialization.py", line 211, in __init__
    super(_open_file, self).__init__(open(name, mode))
FileNotFoundError: [Errno 2] No such file or directory: 'models/pretrained_G_step225000'
srun: error: ravg1124: task 0: Exited with exit code 1
srun: launch/slurm: _step_signal: Terminating StepId=2754586.0
